pygmt/datasets/tile_map.py

```diff
@@ -117,6 +122,10 @@ def load_tile_map(
       * y            (y) float64 ... -7.081e-10 -7.858e+04 ... -1.996e+07 -2.004e+07
       * x            (x) float64 ... -2.004e+07 -1.996e+07 ... 1.996e+07 2.004e+07
         spatial_ref  int64 ... 0
+    >>> # CRS is set only if rioxarray is available
+    >>> if hasattr(raster, "rio"):
+    ...     raster.rio.crs
```

Comment:
To be honest, we probably don't need the `if hasattr(raster, "rio")` check, because the `spatial_ref   int64 ... 0` line above (added in #3321) already means that rioxarray is used (the doctests CI run with `rioxarray` installed). But I guess it'll be good to mention `rioxarray` somewhere in the docstring, albeit indirectly.